src/test/java/org/openrewrite/java/logging/slf4j/CompleteExceptionLoggingTest.java

```diff
@@ -16,7 +16,7 @@
 package org.openrewrite.java.logging.slf4j;
 
 import org.junit.jupiter.api.Test;
-import org.openrewrite.internal.DocumentExample;
```

Comment:
I was unable to build locally without this change, not sure if I had to do some additional setup or this somehow sneaked in with a variable version of some dependency.

Reply:
This is fine; we recently moved that class as it's opened up to external use as well; https://github.com/openrewrite/rewrite/commit/603339be9e82f049e3472eacf6bd49ff4ac456a9

What it does is highlight once example test as the one that we want to show up in the documentation.
So while it's not visible yet, this annotation will make it show up in the documentation next time around:
https://docs.openrewrite.org/recipes/java/logging/slf4j/slf4jlogshouldbeconstant